README.md

```diff
@@ -157,23 +157,27 @@ If updating your environment variable doesn't work, try `rm -rf _build` and then
 
 The Wagtail cms allows for a maximum file size of 10MB.
 
-Althouth I would suggest that files be no bigger than 1MB.
+We would suggest that files be no bigger than 1MB.
 
-Be aware though that many uploads of high resolution images will result in increased costs from our file hosting provider.
+If the image size is too large, users on poor connections or on mobile data will experience slow performance and increased data download costs.
 
 ##### Resolution
 
 Be aware that if an image has too high a resolution, clients will have to download more onto their device causing pages to take longer to load.
 
 If an image has a too low resolution, then the image could be pixilated on high resolution displays.
 
+If possible, please test your proposed image on the [site's staging area](https://ldmw-cms-staging.herokuapp.com/admin) so that you can determine how this will look
+
 ##### Preferred file type
 
 Wagtail facilitates GIF, JPEG, PNG, but doesn't specify a preferred file type (you should go for png if you have the choice though)
 
 ##### Hero background image
 
-The image should contrasts well with white text on top of it
+The image should contrasts well with white text on top of it.
+
+The image should be a wide, with a dimention ratio of approximately 3x2, try to aim for a pixel ratio between 500px and 1200px.
 
-The image should be a wide, with a dimention ratio of approximately 3x2, try to aim for a pixel ratio between 500px and 1200px
+Also note that on mobile the hero image is cut off so please test your image on the [staging area](https://ldmw-cms-staging.herokuapp.com/admin)
```

Comment:
@shouston3 As discussed, let's use different terminology here to explain it is working as expected and not a bug - something along the lines of using a one size fits all image means it is displayed in different ways on each form factor as we have specified that it needs to cover the full background of the text it contains.

Let's also:
+ Add an example screenshot to the documentation 
+ Centre the image on mobile